daphne_server/src/router/mod.rs

```diff
@@ -445,7 +445,7 @@ mod test {
                 .uri("/v02/parse-version")
                 .body({
                     let mut v = Vec::new();
-                    task_id.encode(&mut v);
+                    task_id.encode(&mut v).expect("XXX");
```

Comment:
@mendess how should we handle this encoding failure? `.unwrap()` is probably fine here since `Id::encode()` is actually infallible.

Reply:
prefer unwrapping in tests if you don't expect the call to fail. Makes the tests more useful as they are more likely to panic in the event of code changes